packages/core/index.d.ts

```diff
@@ -99,6 +99,7 @@ declare module '@rjsf/core' {
         > {
         id: string;
         schema: JSONSchema7;
+        uiSchema: UiSchema;
```

Comment:
Can you update documentation to include uiSchema as a prop passed to all widgets?

Reply:
@epicfaace just pushed, I think I added it to the right place. 🤞